Articles/endpoint-viewer-wpf-status-bar.md

```diff
@@ -0,0 +1,60 @@
+# EndpointViewerWPF Status Bar
+Before you can start editing the status bar, you need to add two assembly references. To do this, right-click on your solution and select `Add -> Reference...`. Here, you need to add the `ProconTel.IoC.dll` and `ProconTel.Extensions.Services.dll` by using the bottom `Browse...` button. You can find these dlls in your ProconTel installation directory in the `Client` folder.
```

Comment:
Change to:
Before you can start editing the status bar, you need to add two assembly references `ProconTel.IoC.dll` and `ProconTel.Extensions.Services.dll` to your project. You can find these dlls in your ProconTel installation directory in the `Client` folder.